includes/class-boldgrid-backup.php

```diff
@@ -127,6 +127,11 @@ private function load_dependencies() {
 		 */
 		require_once BOLDGRID_BACKUP_PATH . '/admin/class-boldgrid-backup-admin-notice.php';
 
+		/**
```

Comment:
We may not need this comment.

If you want to keep it, we can change it from:

```php
		/**
		 * The class responsible for auto updates.
		 */
```

... to ...

```php
// The class responsible for auto updates.
```

For single line comments, just use `//``.

Reply:
resolved